patch-package
--- a/node_modules/avvio/boot.js
+++ b/node_modules/avvio/boot.js
@@ -185,7 +185,9 @@ Boot.prototype._addPlugin = function (plugin, opts, callback) {
   callback = callback || null
 
   if (this.booted) {
-    throw new Error('root plugin has already booted')
+    if (process.env.NODE_ENV === 'production') { // hotfix by teil to make route re-loading work in development
+      throw new Error('root plugin has already booted')
+    }
   }
 
   // we always add plugins to load at the current element
